clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/VenicePushJobTest.java

```diff
@@ -0,0 +1,156 @@
+package com.linkedin.venice.hadoop;
+
+import static com.linkedin.venice.hadoop.VenicePushJob.D2_ZK_HOSTS_PREFIX;
+import static com.linkedin.venice.hadoop.VenicePushJob.INCREMENTAL_PUSH;
+import static com.linkedin.venice.hadoop.VenicePushJob.KEY_FIELD_PROP;
+import static com.linkedin.venice.hadoop.VenicePushJob.LEGACY_AVRO_KEY_FIELD_PROP;
+import static com.linkedin.venice.hadoop.VenicePushJob.LEGACY_AVRO_VALUE_FIELD_PROP;
+import static com.linkedin.venice.hadoop.VenicePushJob.MULTI_REGION;
+import static com.linkedin.venice.hadoop.VenicePushJob.PARENT_CONTROLLER_REGION_NAME;
+import static com.linkedin.venice.hadoop.VenicePushJob.PushJobSetting;
+import static com.linkedin.venice.hadoop.VenicePushJob.SOURCE_ETL;
+import static com.linkedin.venice.hadoop.VenicePushJob.SOURCE_KAFKA;
+import static com.linkedin.venice.hadoop.VenicePushJob.StoreSetting;
+import static com.linkedin.venice.hadoop.VenicePushJob.VALUE_FIELD_PROP;
+import static com.linkedin.venice.hadoop.VenicePushJob.VENICE_DISCOVER_URL_PROP;
+import static com.linkedin.venice.hadoop.VenicePushJob.VENICE_STORE_NAME_PROP;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import com.linkedin.venice.compression.CompressionStrategy;
+import com.linkedin.venice.exceptions.UndefinedPropertyException;
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.utils.VeniceProperties;
+import java.util.Properties;
+import java.util.Set;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/**
+ * Unit tests for VenicePushJob class.
+ */
+public class VenicePushJobTest {
```

Comment:
I think you are looking for `TestVenicePushJobConfig`. Sorry for the bad naming 😂 but the good name has picked by the integ test suite

Reply:
I think that would be great. Can you also add some notes on both `VenicePushJobTest` and `TestVenicePushJob` so people would know which is for integ and which is for unit test?